### PR TITLE
feat: add http proxy support when executing package managers

### LINKED ISSUE #80: [REQ]  Add http proxy support when executing package managers
### What kind of request is this?

Improvement of existing experience

### What is your request or suggestion?

I would like to add http proxy support when executing package manager updates, so we can use this tool in environments using corporate proxies.

### PR DESCRIPTION
**Pull request by derdanne**
Add http proxy support when executing package managers.

Closes #80

### CODE REVIEW

File: pkg/utils/utils.go [hunk 1]
```diff
@@ -36,3 +38,22 @@ func IsNonEmptyFile(dir, file string) bool {
 	}
 	return !info.IsDir() && info.Size() > 0
 }
+
```
Comment (sozercan) [inline pkg/utils/utils.go:41]: thanks for the PR! Can we add unit tests for these please?

Comment (derdanne) [reply pkg/utils/utils.go:41]: Done!

**Review by sozercan — APPROVED** [review]
Thanks! LGTM

**Comment by sozercan** [comment]
One of the tests are failing, but I don't think this is related to this. Investigating.

**Pull request merged**